.changeset/five-trains-stare.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-entity-feedback-backend': patch
+'@backstage/plugin-entity-feedback-common': patch
+'@backstage/plugin-entity-feedback': patch
+---
+
+Allow application rating using the `entity-feedback` plugin
+
+See `/plugins/entity-feedback/README.md` for more information
```

Comment:
Let's use a link to NPM or the Backstage repo here so that it's easier to reach for readers